session.go

```diff
@@ -1283,8 +1274,7 @@ const (
 // a MongoDB user within the db database. If the named user doesn't exist
 // it will be created.
 //
-// This method should only be used from MongoDB 2.4 and on. For older
-// MongoDB releases, use the obsolete AddUser method instead.
+// This method should only be used from MongoDB 2.4 and on.
```

Comment:
Nitpick: 
If we are removing support for Mongo <= 2.6 then this line can also be removed I think. 
In my opinion there is no point in mentioning the unsupported Mongo versions in other places than the readme.